src/plugins/iconv/iconv.c

```diff
@@ -26,6 +26,11 @@
 
 
 #include "iconv.h"
+#include <stdio.h>
+#include <stdlib.h>
+#include <errno.h>
+
+#define CHAR_SIZE_MAX 4
```

Comment:
the wides character encoding i know is 4 bytes long. the OS should deal with the rest of the buffering.